lms/lms/lib.rs

```diff
@@ -0,0 +1,14 @@
+//! LMS in Rust
```

Comment:
FWIW, pretty much all of our crates support `no_std`, though I'm happy to address that as a followup

Reply:
I think our code is `no_std` compatible but it has not been explicitly verified. We can probably address this some time later.